spec/mumukit/uri_spec.rb

```diff
@@ -19,13 +19,12 @@ def add_extra_path(original_url)
     it { expect(add_extra_subdomain('http://foo.bar.com/foo?z=3')).to eq 'http://extra.foo.bar.com/foo?z=3' }
     it { expect(add_extra_subdomain('http://foo.bar.com:3000/foo?z=3')).to eq 'http://extra.foo.bar.com:3000/foo?z=3' }
   end
-  describe '#subroute' do
+  describe '#tenantize' do
     it { expect(add_extra_path('https://foo.bar')).to eq 'https://foo.bar/extra/' }
     it { expect(add_extra_path('http://foo.bar')).to eq 'http://foo.bar/extra/' }
-    it { expect(add_extra_path('http://foo.bar/zaraza')).to eq 'http://foo.bar/extra/zaraza' }
+    it { expect(add_extra_path('http://foo.bar/zaraza')).to eq 'http://foo.bar/zaraza/extra/' }
 
     it { expect(add_extra_path('http://foo.bar.com/')).to eq 'http://foo.bar.com/extra/' }
-    it { expect(add_extra_path('http://foo.bar.com/foo?z=3')).to eq 'http://foo.bar.com/extra/foo?z=3' }
-    it { expect(add_extra_path('http://foo.bar.com:3000/foo?z=3')).to eq 'http://foo.bar.com:3000/extra/foo?z=3' }
```

Comment:
I'm not testing this nor supporting it as this method is used only for adding a tenant for base_url.

Reply:
:thinking: I am wondering that may introduce subtle issues in the future

Reply:
Perhaps if we don't support that, we should raise an exception. We have had already some bugs that we causing that improper url building based on implicit preconditions that weren't match.

Reply:
Well, we just tested this and although a '/' is appended to the url, url params still work! so I'm just readding those tests